docs/config.json

```diff
@@ -260,6 +256,20 @@
             }
           ]
         },
+        {
+          "title": "JWT Authentication",
```

Comment:
I like the idea of adding a section at this level, especially if the plan is to include more guides that use the JWT approach to register apps, but I think we should rename the "Guides" section so it's clearer how the two sections differ. 

Is there something that the guides in "Guides" share that differs from the JWT-specific guides? It seems like they all involve adding apps without configuring header rewrites, for example—is there a concise way to express that?

Reply:
@ptgott How about `Access Guides` and `JWT Guides`?

Reply:
I think that would work. I think it would also help to:
- Separate "Access Guides" and "JWT Guides" in `/docs/application-access/introduction/`
- Add a short paragraph at the top of `/docs/application-access/jwt/` explaining this set of guides